services/token-reward-service/README_NEW.md

```
@@ -0,0 +1,266 @@
+Token Reward Service là một microservice quản lý token rewards, deposit và withdrawal cho hệ thống học tập. Service này tích hợp với blockchain Ethereum để xử lý token transactions và database PostgreSQL để lưu trữ dữ liệu.
+
+## Kiến trúc
+
+- **Node.js** + **Express.js** - Web framework
+- **Sequelize** + **PostgreSQL** - Database ORM và DBMS
+- **Ethers.js** - Blockchain interaction
+- **WebSocket** - Real-time blockchain event listening
+
+## Cấu trúc thư mục
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)